docs/system-admin-guide/integrations/nextcloud/README.md

```diff
@@ -1,79 +1,96 @@
 
-![Navigation_link_OpenProject](Navigation_link_OpenProject.png)
+To activate the module, go to the **Project settings** in the desired project and access the **Modules** entry in the side menu. There, activate the *File storages* module. Once this is done, a new menu entry called **File storages** appears at the bottom of the side menu, where you can select your new Nextcloud integration.
 
-By activating "enable unified search for tickets" in your personal settings, the Nextcloud dashboard will include OpenProject information in the the built-in universal search:
+![module_activation](Settings_modules.png)
 
-![Unified_search](Unified_search.png)
+![settings_files_storages](Settings_files_storages.png)
 
-**Set up of OAuth to OpenProject**
+## How to reset your OAuth values
 
-Within your Settings under Administration and then Connected Accounts you can set-up the OAuth authentication to your OpenProject instance.
+If you need to reset the values of the Nextcloud integration (as an administrator), you can always reset the OAuth values from both sides of the integration by clicking on the **Reset OAuth values** (in Nextcloud) or **Replace OAuth values** (in OpenProject) buttons.
 
-![OAuth](OAuth.png)
+> **Important**: When you reset/replace these values, you will need to update the configuration with the new OAuth credentials from the side you are reseting. This will also require all users to re-authorize OpenProject to access their Nextcloud account by logging in again.
 
-In OpenProject, add Nextcloud as application under Administration then Authentication and OAuth and enter the information in your Nextcloud instance.
+![nextcloud_reset_OPOAuth](3_2_06 -NC_OP_OAuth_Replace.png)
 
-![OpenProject_OAuth](OpenProject_OAuth.png)
+![nextcloud_reset_NCOAuth](3_2_07-NC__OAuth_Replace.png)
 
-## Where do I find the Nextcloud integration in OpenProject?
+![openproject_reset_OAuth](3_4_03-OP_Replace_Alert.png)
 
-Further integration efforts are under way, which will deliver a Nextcloud integration also on the OpenProject side.
 
-## What if project notifications are not displayed?
+## How to delete a file storage integration
 
-If the notifications are not displayed in your Nextcloud dashboard, please check the following in your Nextcloud basic settings: in the background jobs, Cron must be activated.
+As an administrator, you can always delete the integration to Nextcloud using the **Delete** button in OpenProject settings or the **Reset** button in Nextcloud settings.
 
-![Cron_job_settings](Cron_job_settings.png)
\ No newline at end of file
+> **Important:** If you perform this action, the integration will be reset and deleted and all settings and user connections that were created will be deleted. This means that should you want to reconfigure the integration, you will need complete the entire setup process once again.
```

Comment:
@marcalcobe Hmm, the delete in OpenProject deletes all file links. The reset button in Nextcloud just deletes the connections but not the links. We may want to be very clear what does what. So we better invest these lines here.

Reply:
@wielinde totally agree with this quote add. Thanks.